wiki/Tournaments/SAS/1/en.md

```diff
@@ -46,17 +46,324 @@ The Score Attack Showdown is run by various community members.
+| #2,621 | ![][flag_BE] [flyerbram002](https://osu.ppy.sh/users/12725421) |
+| #2,728 | ![][flag_BR] [Namirin-chan](https://osu.ppy.sh/users/6761832) |
+| #2,859 | ![][flag_SG] [IHxveNoWuhxn](https://osu.ppy.sh/users/15210942) |
+| #2,871 | ![][flag_ID] [Soraru Aoki](https://osu.ppy.sh/users/15636932) |
+| #2,933 | ![][flag_ID] [HerosonRhymes](https://osu.ppy.sh/users/12640004) |
+| #3,009 | ![][flag_FI] [Stiffu](https://osu.ppy.sh/users/12274556) |
+| #3,283 | ![][flag_DO] [Antalf](https://osu.ppy.sh/users/8793773) |
+| #3,295 | ![][flag_ID] [Fr05tyGD](https://osu.ppy.sh/users/14230684) |
+| #3,322 | ![][flag_MY] [Unicow](https://osu.ppy.sh/users/9444174) |
+| #3,354 | ![][flag_BR] [Yuragi](https://osu.ppy.sh/users/8501380) |
+| #3,430 | ![][flag_AR] [NEDEAAAHHHH](https://osu.ppy.sh/users/65724) |
+| #3,530 | ![][flag_FR] [KewaiiGames](https://osu.ppy.sh/users/14797411) |
+| #3,600 | ![][flag_MY] [aswdafxc](https://osu.ppy.sh/users/12054898) |
+| #3,613 | ![][flag_US] [3lijah](https://osu.ppy.sh/users/4686148) |
+| #3,654 | ![][flag_US] [\[LS\]Treeman](https://osu.ppy.sh/users/14677848) |
+| #3,678 | ![][flag_VE] [Neko-Pan](https://osu.ppy.sh/users/8676532) |
+| #3,685 | ![][flag_PH] [Edison](https://osu.ppy.sh/users/1325159) |
+| #3,824 | ![][flag_US] [Feylos](https://osu.ppy.sh/users/4565040) |
+| #3,931 | ![][flag_US] [Ninty](https://osu.ppy.sh/users/11631179) |
+| #3,938 | ![][flag_BR] [AlgocomB](https://osu.ppy.sh/users/12298887) |
+| #3,975 | ![][flag_NZ] [- Abby -](https://osu.ppy.sh/users/7740118) |
+| #4,024 | ![][flag_ES] [game\_maker26](https://osu.ppy.sh/users/4876699) |
+| #4,106 | ![][flag_CL] [-JustLenny](https://osu.ppy.sh/users/12793583) |
+| #4,215 | ![][flag_KR] [S ILY](https://osu.ppy.sh/users/9766317) |
+| #4,323 | ![][flag_US] [\[LS\]Crazy](https://osu.ppy.sh/users/12211556) |
+| #4,503 | ![][flag_PL] [Mr\_adamello](https://osu.ppy.sh/users/7420894) |
+| #4,870 | ![][flag_AR] [boligoma](https://osu.ppy.sh/users/13310563) |
+| #5,017 | ![][flag_US] [walnutt](https://osu.ppy.sh/users/8617859) |
+| #5,331 | ![][flag_MX] [\[LS\] Shuri](https://osu.ppy.sh/users/14593518) |
+| #5,546 | ![][flag_NO] [prebert123](https://osu.ppy.sh/users/7859738) |
+| #5,607 | ![][flag_TW] [Joylintp](https://osu.ppy.sh/users/13579078) |
+| #5,732 | ![][flag_US] [FlynnMac](https://osu.ppy.sh/users/13857986) |
+| #6,162 | ![][flag_AU] [\[LG\]aurusei](https://osu.ppy.sh/users/14430622) |
+| #6,748 | ![][flag_BR] [Halo-](https://osu.ppy.sh/users/7572142) |
+
+*Note: Global rank number was taken in the end of registration.*
 
 ## Mappools
 
-*Mappools will be listed once the Round 1 mappool has been released.*
+### Round 2
+
+**[Download the mappack here! (27 MB)](https://cdn.discordapp.com/attachments/711778005399109687/772567150929641502/SAS_Round_2.zip)**
+
+- Hybrid
+  - [plusol - Angel Temple (BringoBrango) \[Ascension (SAS edit)\]](https://osu.ppy.sh/beatmapsets/1286284#mania/2670767)
```

Comment:
might as well not be a second list if it's only one map. id do something like `- Hybrid: [plusol - ...` for each of these